backend/routes/rankings.js

```diff
@@ -1,13 +1,13 @@
 import express from "express";
-import { RankingV2 } from "../ranking.js";
+import { RankingV3 } from "../ranking.js";
```

Comment:
Great to see the benefits of encapsulation, the mechanism is substituted, and everything else keeps working 👍